### ISSUE TITLE
Workflow BD Topo - 44109

### ISSUE THREAD

**Issue by gpetit** [orbisgis/geoclimate#304]
I'm trying to run [this configuration file](https://github.com/orbisgis/geoclimate/blob/master/processingchain/src/test/resources/org/orbisgis/orbisprocess/geoclimate/processingchain/config/bdtopo_workflow_folderinput_folderoutput_id_zones.json) for BD Topo v2 on the city of Nantes (44109).
My input shapefiles are here `/home/gpetit/geoclimate/bdtopo/input/44/`

So in my configuration file I wrote
```json
"input" : {
        "folder": {"path" :"/home/gpetit/geoclimate/bdtopo/input/44/",
            "id_zones":["44109"]}
    }
```

And when I executed the chain in DBeaver I obtain this erro logs

```
!ENTRY groovy.lang.GroovyShell 1 0 2020-03-04 10:51:44.291
!MESSAGE Reading file parameters from /home/gpetit/geoclimate/bdtopo/config_id_zone.json

!ENTRY groovy.lang.GroovyShell 1 0 2020-03-04 10:51:44.293
!MESSAGE Process BD Topo v2 on the city of Nantes

!ENTRY groovy.lang.GroovyShell 4 0 2020-03-04 10:51:44.608
!MESSAGE The iris_ge file doesn't contains any zone identifiers

!ENTRY groovy.lang.GroovyShell 4 0 2020-03-04 10:51:44.610
!MESSAGE Cannot load the files from the folder [path:/home/gpetit/geoclimate/bdtopo/input/44/, id_zones:[44109]]

!ENTRY groovy.lang.GroovyShell 1 0 2020-03-04 10:51:44.613
!MESSAGE null

!ENTRY org.orbisgis.ui.editors.groovy.GroovyJob 1 0 2020-03-04 10:51:44.614
!MESSAGE Groovy script successfully executed.

```

Below is a screenshot of the `IRIS_GE.shp` layer

![image](https://user-images.githubusercontent.com/1871248/75867490-fb784780-5e06-11ea-83cd-d8f14b55a352.png)

Any idea ?

**Comment by gpetit** [comment]
The problem may in this section ? 
https://github.com/orbisgis/geoclimate/blob/1bdf7dabd74060617f3df9807a5944da5c7df8e7/processingchain/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/processingchain/Workflow.groovy#L728

Seems that the INSEE_COM column is not recognized ?

https://github.com/orbisgis/geoclimate/blob/1bdf7dabd74060617f3df9807a5944da5c7df8e7/processingchain/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/processingchain/Workflow.groovy#L749
Would it be `id_zones = findIDZones(h2gis_datasource, insee_com)` instead ?

@ebocher

**Comment by gpetit** [comment]
I also tried with 
- `"id_zones":"44109"` (without [ ]): same error
-  `"id_zones":['44109']` : no error ... just stop the process instantly

**Comment by gpetit** [comment]
I should miss something somewhere ...
I tried with [this configuration](https://github.com/orbisgis/geoclimate/blob/master/processingchain/src/test/resources/org/orbisgis/orbisprocess/geoclimate/processingchain/config/bdtopo_workflow_dbinput_dboutput.json) file (db to db)

```json
"input" : {
        "database": {
            "user": "xxxxx",
            "password": "xxxx",
            "url": "jdbc:postgresql://xxxxx:5432/paendora",
            "id_zones":["44109"],
            "tables": {"iris_ge":"ign_iris.iris_ge_2016",
                "bati_indifferencie":"ign_bdtopo_2017.bati_indifferencie",
                "bati_industriel":"ign_bdtopo_2017.bati_industriel",
                "bati_remarquable":"ign_bdtopo_2017.bati_remarquable",
                "route":"ign_bdtopo_2017.route",
                "troncon_voie_ferree":"ign_bdtopo_2017.troncon_voie_ferree",
                "surface_eau":"ign_bdtopo_2017.surface_eau",
                "zone_vegetation":"ign_bdtopo_2017.zone_vegetation",
                "terrain_sport":"ign_bdtopo_2017.terrain_sport",
                "construction_surfacique":"ign_bdtopo_2017.construction_surfacique",
                "surface_route":"ign_bdtopo_2017.surface_route",
                "surface_activite":"ign_bdtopo_2017.surface_activite"}
        }
```
And I obtain

```
!ENTRY groovy.lang.GroovyShell 1 0 2020-03-05 10:17:41.957
!MESSAGE Reading file parameters from /home/gpetit/geoclimate/bdtopo/config_bd_to_bd.json

!ENTRY groovy.lang.GroovyShell 1 0 2020-03-05 10:17:41.959
!MESSAGE Example of configuration file to load data from a database and save the result into the same database

!ENTRY groovy.lang.GroovyShell 1 0 2020-03-05 10:17:41.976
!MESSAGE null

!ENTRY org.orbisgis.ui.editors.groovy.GroovyJob 1 0 2020-03-05 10:17:41.977
!MESSAGE Groovy script successfully executed.
```

**Comment by gpetit** [comment]
We investigated with @SPalominos and seems the problem come from the `.SHP` extension which is in uppercase, while OrbisData was only able to load driver when extensions are in lowercase.

```
INFO] org.orbisgis.orbisprocess.geoclimate.processingchain.ProcessingChain - Loading file /home/gpetit/geoclimate/bdtopo/input/44/SURFACE_ACTIVITE.SHP 0 on 11
[ERROR] org.orbisgis.orbisdata.datamanager.jdbc.io.IOMethods - Unsupported file format.
Supported formats are : [shp, geojson, tsv, csv, dbf, kml, kmz, osm, gz, bz, gpx].
```

**Comment by gpetit** [comment]
Resolved with orbisgis/orbisdata#218 !
